### PR TITLE
Assignment 6 #117955

### PR DESCRIPTION
**Pull request by rounak-successive**

### CODE REVIEW

File: Assignment-Day-6/Question2.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+class Shape {
+  constructor(width, height) {
+    this.width = width
+    this.height = height
+  }
+
+  getArea() {
+    return NaN
+  }
```
Comment (Vishal-successive) [inline Assignment-Day-6/Question2.js:11]: why return NaN?

File: Assignment-Day-6/Question2.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+}
+
+class Rectangle extends Shape {
+  constructor(width, height) {
+    super(width, height)
+  }
+
+  getArea() {
+    return this.width * this.height
```
Comment (Vishal-successive) [inline Assignment-Day-6/Question2.js]: write logic in getArea() of Shape class and use it here

File: Assignment-Day-6/Question2.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+}
+
+class Triangle extends Shape {
+  constructor(width, height) {
+    super(width, height)
+  }
+
+  getArea() {
+    return (this.width * this.height) / 2
+  }
```
Comment (Vishal-successive) [inline Assignment-Day-6/Question2.js:33]: write logic in getArea() of Shape class and use it here

File: Assignment-Day-6/Question2.js (hunk 1 above)
Comment (rounak-successive) [reply Assignment-Day-6/Question2.js:11]: I had added NaN as we can return the area only when we know what type of shape the calling object.

**Comment by Vishal-successive** [comment]
looks good :+1:

**Pull request merged**